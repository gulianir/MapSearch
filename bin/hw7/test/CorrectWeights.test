LoadGraph hp harryPotter.tsv

ListChildren hp Harry
ListChildren hp Ron
ListChildren hp Dementors
ListChildren hp Voldemort