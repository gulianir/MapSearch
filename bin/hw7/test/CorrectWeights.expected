loaded graph hp

the children of Harry in hp are: Dementors(1.000) Ron(0.333) Voldemort(1.000)
the children of Ron in hp are: Dementors(1.000) Harry(0.333) Voldemort(1.000)
the children of Dementors in hp are: Harry(1.000) Ron(1.000)
the children of Voldemort in hp are: Harry(1.000) Ron(1.000)